sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -32,6 +30,7 @@
 
     @Override
     public DiscountDto getById(Long id) {
+        discountDAO.incrementViewsByDiscountId(id);
```

Comment:
As for my opinion it is better to have separate endpoint to increase views number. Otherwise we should rename method to something like `incrementViewsAndGet`

Reply:
Separated endipoint is created.